### DISCUSSION TITLE
Is there a function / command / shortcut to open data viewer of the variable.

### DISCUSSION THREAD

**Discussion by michaelmiscanuk** [microsoft/vscode-jupyter#9850] [Questions and Answers] [answered]
Hi.

I refer to this functionality in VSCode Jupyter:
https://code.visualstudio.com/docs/python/jupyter-support-py#_variables-explorer-and-data-viewer

It is quite long to have to open variables - search for variable to display, then display it in data viewer. I would like to ask, if there is some shorcut for it - like a function similar to RStudio View()

Thank you,.

**Comment by DonJayamanne** [comment, marked as answer]
Unfortunately we don't have such a feature, please upvote this existing issue here https://github.com/microsoft/vscode-jupyter/issues/7427
